index.js

```diff
@@ -45,10 +47,15 @@ function fileComment(inputFile, showFileName) {
         changes.filter(obj => obj.change.actions[0] === "update").length + ' to change, ' +
         changes.filter(obj => obj.change.actions[0] === "delete").length + ' to destroy.</b>'
 
+    let openDetails = ""
+    if (expandDetailsComment === true) {
+        openDetails = "open"
+    }
+
```

Comment:
I think you had something like this before?
```suggestion
    const openDetails = expandDetailsComment ? "open" : ""
```